UsbIpServer/Server.cs

```diff
@@ -45,6 +45,11 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
 
             TcpListener.Start();
 
+            if (RegistryUtils.HasRegistryAccess())
+            {
+                RegistryUtils.InitializeRegistry();
```

Comment:
This I don't like. `InitializeRegistry` always starts with a clean slate. The beauty of the registry is that it is persistent and survives reboots. Otherwise you might as well have used IPC to bind/unbind.

My personal use case for usbipd-win is to share a token to a Hyper-V guest (a virtualized Linux server). Currently, I can always claim that at guest boot time (due to usbipd-win sharing all devices). With the explicit bind/unbind I would really like the host to persist any bound devices.

Suggestion: WiX will create the master RegKey and recursively deletes it on uninstall. And we remove this initialization. Then and `--bind` and `--unbind` is persistent.

(alternatively I could alter the service startup arguments to include `--all`, of course)...

Reply:
(and with that last remark I mean: on my personal machine, not for every install)